lib/rules/input-image-alt.json

```
@@ -9,7 +9,9 @@
     "wcag412",
     "section508",
     "section508.22.a",
-    "ACT"
+    "ACT",
+    "TTv5",
+    "TT5.c"
```

ANDI seems to pick image buttons up as images, so I think this fits better under 7.a

```suggestion
    "TT7.a"
```